From db7300245a84623c043c7729729a9f2814ff3dbe Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Thu, 5 Mar 2015 11:59:25 +0800
Subject: [PATCH 284/284] atomisp2: Remove revision checking for ISP2400A0

This patch removes the revision checking for
ISP2400A0 which is not used any more.

Change-Id: Ic66028b1e3842ef9f1eaece936bc46a153a0e52b
Fixes-Issue: #H1503933655
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/44737
Approver: Bin B Han <bin.b.han@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Build: viefandr
Integrator: Wei Tao <wei.tao@intel.com>
Approver: Wei Tao <wei.tao@intel.com>
Build: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 .../camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c   | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c
index e9167c0..154e0ec 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c
@@ -1147,11 +1147,7 @@ static bool is_valid_device(struct pci_dev *dev,
 		return true;
 	}
 
-#ifdef ISP2400
-	return dev->revision <= a0_max_id;
-#else /* ISP2400 */
 	return dev->revision > a0_max_id;
-#endif /* ISP2400 */
 }
 
 static int init_atomisp_wdts(struct atomisp_device *isp)
-- 
1.9.1

